### PR TITLE
[[ Bug 22940 ]] Remove support for 32bit on MacOS IDE and standalones

### PR DESCRIPTION
**Pull request by livecodepanos**
WIP until a PR that removes support for 32bit MacOS in the S/B is ready

### CODE REVIEW

**Comment by livecodepanos** [comment]
Goes together with https://github.com/livecode/livecode-ide/pull/2144

File: ide-support/revsaveasstandalone.livecodescript [hunk 1]
```diff
@@ -660,7 +665,11 @@ command revSaveAsMacStandalone pStack, pRestoreFileName, pFolder, @xStandalonePa
    
    -- Architectures
    local tArchitectures
-   if sStandaloneSettingsA["MacOSX x86-32"] then put "i386," after tArchitectures
+   if sStandaloneSettingsA["MacOSX x86-32"] then 
+      delete variable sStandaloneSettingsA["MacOSX x86-32"]
+      put true into sStandaloneSettingsA["MacOSX x86-64"]
+   end if
```
Comment (livecodeian) [inline ide-support/revsaveasstandalone.livecodescript]: @livecodepanos do we need this? It looks like this check is already performed in revSaveAsStandalone

Comment (livecodepanos) [reply ide-support/revsaveasstandalone.livecodescript]: @livecodeian Indeed this was not needed, I have now removed it. Thanks :)

**Comment by livecodeian** [comment]
@livecode-vulcan review ok baaead1

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeian ok baaead154a98ab08f6c13175914e1c4d31f2f8b0

**Comment by livecode-vulcan** [comment]
😎 test success baaead154a98ab08f6c13175914e1c4d31f2f8b0

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/893)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/302)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/302)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/302)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/302)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2419)
* try-community-universal-ios-iphoneos14.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.0/builds/21)
* try-community-universal-ios-iphoneos13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.5/builds/72)
* try-community-universal-ios-iphonesimulator14.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.0/builds/22)
* try-community-universal-ios-iphonesimulator13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.5/builds/68)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2446)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2435)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2434)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2469)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2409)

**Pull request merged**